pkg/upgrade/upgrades/sampled_stmt_diagnostics_requests.go

```diff
@@ -48,6 +50,23 @@ func sampledStmtDiagReqsMigration(
 			query:          addSamplingProbColToStmtDiagReqs,
 			schemaExistsFn: hasColumn,
 		},
+		{
+			name:       "drop-stmt-diag-reqs-v1-index",
+			schemaList: []string{"completed_idx"},
+			query:      dropCompletedIdxV1,
+			schemaExistsFn: func(existing, _ catalog.TableDescriptor, _ string) (bool, error) {
+				idx, err := existing.FindIndexWithName("completed_idx")
+				if err != nil { // if the index doesn't exit, we're good
+					err = nil
+					return true, err // defeat the linter about returning a nil err
+				}
```

Comment:
Alternatively, this might also defeat the linter:
```
idx, _ := existing.FindIndexWithName("completed_idx")
if idx == nil {
  return true, nil
}
```